apps/template/src/App.tsx

```diff
@@ -12,7 +12,7 @@ import { ErrorBoundary } from "react-error-boundary"
 interface AppProps {
   theme?: string
   embedded?: string | boolean
-  endpoint?: string
+  endpoint?: string | null | undefined
```

Comment:
I think there is no need for an explicit `undefined` because `?` makes a property optional which already covers that case.

Reply:
agree!